src/Microsoft.AspNetCore.Authentication.Facebook/FacebookOptions.cs

```diff
@@ -57,6 +60,6 @@ public string AppSecret
         /// The list of fields to retrieve from the UserInformationEndpoint.
         /// https://developers.facebook.com/docs/graph-api/reference/user
         /// </summary>
-        public IList<string> Fields { get; }
+        public ICollection<string> Fields { get; } = new HashSet<string>();
```

Comment:
Nice! :smile: 

Should we also update `OAuthOptions.Scope`/`OpenIdConnectOptions.Scope` to use a set instead of a good old list? (and why not using `ISet<string>`?)